runtime/kusama/src/lib.rs

```diff
@@ -1098,20 +1101,55 @@ impl parachains_paras::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type WeightInfo = weights::runtime_parachains_paras::WeightInfo<Runtime>;
 	type UnsignedPriority = ParasUnsignedPriority;
+	type UmpQueueTracker = ParaInclusion;
 	type NextSessionRotation = Babe;
 }
 
 parameter_types! {
-	pub const FirstMessageFactorPercent: u64 = 100;
+	/// Amount of weight that can be spent per block to service messages.
+	///
+	/// # WARNING
+	///
+	/// This is not a good value for para-chains since the `Scheduler` already uses up to 80% block weight.
+	pub MessageQueueServiceWeight: Weight = Perbill::from_percent(40) * BlockWeights::get().max_block;
```

Comment:
scheduler should get up to 100% of the block weight really, since it's a service used primarily for real-(block)time system tasks. MQ is used to execute non-real-time user-related tasks, so ideally it would get maybe 50% of what's left after scheduler has finished.